sys-block/gpart/gpart-0.3.recipe

```diff
@@ -0,0 +1,49 @@
+SUMMARY="A tool for recovering MBR partition tables in hard disks"
+DESCRIPTION="Gpart is a small tool which tries to guess what partitions are \
+on a PC type, MBR-partitioned hard disk in case the primary partition table \
+was damaged."
+HOMEPAGE="https://github.com/baruch/gpart"
+COPYRIGHT="1999-2017 Michail Brzitwa"
+LICENSE="GNU GPL v2"
+REVISION="1"
+srcGitRev="23b651d1e9c6ae1454e5d3fe315d7d4e145661c0"
+SOURCE_URI="https://github.com/baruch/gpart/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="c492b7dc677e4c6af197f9598ba815a4e14a03578ee8a963430529767bc6c947"
+SOURCE_FILENAME="gpart-$portVersion-$srcGitRev.tar.gz"
+SOURCE_DIR="gpart-$srcGitRev"
+PATCHES="gpart-$portVersion.patchset"
+
+ARCHITECTURES="x86_gcc2 ?x86 x86_64"
+
+PROVIDES="
+	gpart = $portVersion
+	cmd:gpart = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+
+BULID_PREREQUIRES="
+	haiku_devel
+	"
+BUILD_REQUIRES="
```

Comment:
You swaped `BUILD_REQUIRES` and `BUILD_PREREQUIRES`. If you look at [my previous comment](https://github.com/haikuports/haikuports/pull/2059/#discussion_r160011209), `BUILD_PREREQUIRES` comes _after_ `BUILD_REQUIRES`, not _before_.

Here's the new diff to correct this:
```diff
-BUILD_PREREQUIRES="
+BUILD_REQUIRES="
 	haiku_devel
 	"
-BUILD_REQUIRES="
+BUILD_PREREQUIRES="
 	cmd:autoconf
 	cmd:automake
 	cmd:gcc
 	cmd:make
 	"
```